src/library-authoring/data/slice.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-param-reassign */
```

Comment:
nit
```suggestion
// @ts-check
/* eslint-disable no-param-reassign */
```

Reply:
Before each of these PRs merge, please convert to full `.ts`/`.tsx` files, and get rid of all the JS versions of the types. The PR to make that possible should be merging very soon as it's now unblocked.